instrumentation/opentelemetry-instrumentation-opentracing-shim/tests/test_shim.py

```diff
@@ -422,7 +422,7 @@ def test_log_kv(self):
             )
             self.assertEqual(span.unwrap().events[1].attributes["foo"], "bar")
             # Tolerate inaccuracies of less than a microsecond. See Note:
-            # https://open-telemetry.github.io/opentelemetry-python/opentelemetry.ext.opentracing_shim.html
+            # https://open-telemetry.github.io/opentelemetry-python/opentelemetry.instrumentation.opentracing_shim.html
```

Comment:
As a sanity check, I tried both of these URLs, and they both 404 😅 

- https://open-telemetry.github.io/opentelemetry-python/opentelemetry.ext.opentracing_shim.html
- https://open-telemetry.github.io/opentelemetry-python/opentelemetry.instrumentation.opentracing_shim.html

I think it should be https://open-telemetry.github.io/opentelemetry-python/instrumentation/opentracing_shim/opentracing_shim.html

Reply:
Oh good catch. I'm assuming it uses the structure of the `docs` folder to find the page? I'm not too familiar with what `https://open-telemetry.github.io` is. Is this still read-the-docs?